compiler/plc_project/src/build_config.rs

```diff
@@ -34,25 +36,38 @@ pub enum LinkageInfo {
 }
 
 #[derive(Serialize, Deserialize, Debug)]
+#[serde(deny_unknown_fields)]
 pub struct ProjectConfig {
     pub name: String,
     pub files: Vec<PathBuf>,
     #[serde(default)]
     pub compile_type: FormatOption,
     #[serde(default)]
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub output: Option<String>,
     #[serde(default)]
     pub libraries: Vec<LibraryConfig>,
     #[serde(default)]
     pub package_commands: Vec<String>,
+    #[serde(default)]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub version: Option<String>,
+    #[serde(default)]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(alias = "format-version")]
+    #[serde(rename(serialize = "format-version"))]
```

Comment:
I think you can remove this line, the `[serde(alias = "...")]` line should take care of this?

Reply:
You're right - I'll remove it, thanks!